Userland/Libraries/LibGL/SoftwareGLContext.h

```diff
@@ -176,10 +179,7 @@ class SoftwareGLContext : public GLContext {
     FloatVector4 m_current_vertex_tex_coord = { 0.0f, 0.0f, 0.0f, 1.0f };
     FloatVector3 m_current_vertex_normal = { 0.0f, 0.0f, 1.0f };
 
-    Vector<GLVertex, 96> vertex_list;
-    Vector<GLTriangle, 32> triangle_list;
-    Vector<GLTriangle, 32> processed_triangles;
-    Vector<GLVertex> m_clipped_vertices;
+    Vector<SoftGPU::Vertex, 96> vertex_list;
```

Comment:
This should probably be `m_vertex_list` (old code had it wrong as well)

Reply:
done